### PR TITLE
Fix bug #86

### LINKED ISSUE #86: Update PKGBUILD to v1.5.0
PKGBUILD needs to be updated to point to the v1.5.0 release

### PR DESCRIPTION
**Pull request by levkush**
Closed #86

### CODE REVIEW

**Comment by kehoecj** [comment]
Thanks for the PR @levkush - I need to test this on my Arch environment. If everything looks good I'll approve and will merge

**Review by kehoecj — CHANGES_REQUESTED** [review]

File: PKGBUILD
[the remark this answers is not in the document]
Comment (kehoecj) [reply PKGBUILD:1]: @levkush Reminder for the following requested changes to get this MR approved and merged

[the remark this answers is not in the document]
Comment (levkush) [reply PKGBUILD:1]: I added the required changes.

**Review by kehoecj — APPROVED** [review]
LGTM! Thanks for the PR

**Pull request merged**